src/lib/doctrine-dbal-bulk/tests/Flow/Doctrine/Bulk/Tests/MysqlIntegrationTestCase.php

```diff
@@ -4,22 +4,38 @@
 
 namespace Flow\Doctrine\Bulk\Tests;
 
+use Doctrine\DBAL\Configuration;
 use Doctrine\DBAL\DriverManager;
+use Doctrine\DBAL\Logging\Middleware;
 use Flow\Doctrine\Bulk\Tests\Context\DatabaseContext;
+use Flow\Doctrine\Bulk\Tests\Context\ProxyLogger;
 use PHPUnit\Framework\TestCase;
 
 abstract class MysqlIntegrationTestCase extends TestCase
 {
     protected DatabaseContext $databaseContext;
 
+    protected readonly ProxyLogger $logger;
+
     protected function setUp() : void
     {
-        $this->databaseContext = new DatabaseContext(DriverManager::getConnection(['url' => \getenv('MYSQL_DATABASE_URL')]));
+        $this->logger = new ProxyLogger();
+        $this->databaseContext = new DatabaseContext(
+            DriverManager::getConnection(
+                ['url' => \getenv('MYSQL_DATABASE_URL')],
+                (new Configuration())->setMiddlewares([new Middleware($this->logger)])
+            )
+        );
```

Comment:
I think I would extract logger setting into IntegrationTestCase and make all dialects to extend from it.

Reply:
The reason is the `DriverManager::getConnection` here, it's setting up the configuration, which later (in context) cannot be replaced and always uses default one, thus I needed to move this level up.